index.html

```diff
@@ -1004,6 +1006,95 @@ <h3>
           conventions or limitations of the host operating system.
         </p>
       </section>
+      <section>
+        <h3>
+          `protocol_handlers` member
+        </h3>
+        <p>
+          The [=manifest's=] <code><dfn  data-dfn-for="manifest">protocol_handlers</dfn></code> member is an <a>array</a>
+          of <a>protocol handler description</a>s that allows a web application to
+          handle URL protocols.
+        </p>
+        <p class="example">
+          Protocol handlers could, for instance, be used for web app
+          communication where one app directly invokes another and passes data
+          via custom protocol links.
+        </p>
+        <p>
+          How protocol handlers are presented, and how many of them are shown
+          to the user, is at the discretion of the user agent and/or operating
+          system.
+        </p>
+        <p>To <dfn>process the `protocol_handlers` member</dfn>, given [=object=] |json:JSON|, |manifest:ordered map|, and |manifest URL:URL|:</p>
+        <ol>
+          <li>Let <var>processedProtocolHandlers</var> be a new [=list=].
+          </li>
+          <li>[=list/For each=] <var>protocol_handler</var> (<a>protocol handler description</a>):
+            <ol>
+              <li>If |protocol_handler|["protocol"] or
+                <var>protocol_handler</var>["url"] is undefined, [=iteration/continue=].
+              </li>
+              <li>Let (<dfn>normalizedProtocol</dfn>, <dfn>normalizedUrl</dfn>) be the result of running <a data-cite="HTML/system-state#normalize-protocol-handler-parameters">normalize protocol handler parameters</a> with <var>protocol_handler</var>["protocol"], <var>protocol_handler</var>["url"] using <var>manifest URL</var> as the base URL, and [=this=]'s relevant [=environment settings object=]. If the result is failure, [=iteration/continue=].</li>
```

Comment:
```suggestion
              <li>Let (<dfn>normalizedProtocol</dfn>, <dfn>normalizedUrl</dfn>) be the result of running [=normalize protocol handler parameters=] with <var>protocol_handler</var>["protocol"], <var>protocol_handler</var>["url"] using <var>manifest URL</var> as the base URL, and [=this=]'s relevant [=environment settings object=]. If the result is failure, [=iteration/continue=].</li>
```

Reply:
@marcoscaceres just out of curiosity, why is it preferable to have it defined in HTML instead of linking to it with the data-cite? is the data cite incorrect?